ngraph/frontend/onnx_import/src/op/upsample.cpp

```diff
@@ -70,6 +75,73 @@ namespace ngraph
             }
 
             namespace set_1
+            {
+                OutputVector upsample(const onnx_import::Node& node)
+                {
+                    const auto height_scale = node.get_attribute_value<float>("height_scale");
+                    const auto width_scale = node.get_attribute_value<float>("width_scale");
+                    const auto mode = node.get_attribute_value<std::string>("mode", "nearest");
+                    check_mode_support(node, mode, 1);
+
+                    const auto interpolate_mode = convert_mode(mode);
+                    const auto zero_pad = std::vector<size_t>(1, 0);
```

Comment:
0 is a default value so we don't need to specify it.
vector(1, 0) can be confused with vector{1, 0}
```suggestion
                    const auto zero_pad = std::vector<size_t>(1);
```